File: app/src/main/java/com/qfq/tainzhi/videoplayer/bean/DouyuChannelBean.java
package com.qfq.tainzhi.videoplayer.bean;

import java.util.List;

/**
 * Created by muqing on 2019/6/17.
 * Email: qfq61@qq.com
 */
public class DouyuChannelBean {
    private int id;
    private String  name;
    private String src;
    private String url;
    private String icon;
    
    public int getId() {
        return id;
    }
    
    public void setId(int id) {
        this.id = id;
    }
    
    public String getName() {
        return name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public String getSrc() {
        return src;
    }
    
    public void setSrc(String src) {
        this.src = src;
    }
    
    public String getUrl() {
        return url;
    }
    
    public void setUrl(String url) {
        this.url = url;
    }
    
    public String getIcon() {
        return icon;
    }
    
    public void setIcon(String icon) {
        this.icon = icon;
    }
    
    @Override
    public String toString() {
        return "DouyuChannelBean{" +
                       "id=" + id +
                       ", name='" + name + '\'' +
                       ", src='" + src + '\'' +
                       ", url='" + url + '\'' +
                       ", icon=" + icon +
                       '}';
    }
}

File: app/src/main/java/com/qfq/tainzhi/videoplayer/bean/DouyuChannelBean.kt
package com.qfq.tainzhi.videoplayer.bean

/**
 * Created by muqing on 2019/6/17.
 * Email: qfq61@qq.com
 */
class DouyuChannelBean constructor() {
    private var id: Int = 0
    private var name: String? = null
    private var src: String? = null
    private var url: String? = null
    private var icon: String? = null
    fun getId(): Int {
        return id
    }
    
    fun setId(id: Int) {
        this.id = id
    }
    
    fun getName(): String? {
        return name
    }
    
    fun setName(name: String?) {
        this.name = name
    }
    
    fun getSrc(): String? {
        return src
    }
    
    fun setSrc(src: String?) {
        this.src = src
    }
    
    fun getUrl(): String? {
        return url
    }
    
    fun setUrl(url: String?) {
        this.url = url
    }
    
    fun getIcon(): String? {
        return icon
    }
    
    fun setIcon(icon: String?) {
        this.icon = icon
    }
    
    public override fun toString(): String {
        return ("DouyuChannelBean{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", src='" + src + '\'' +
                ", url='" + url + '\'' +
                ", icon=" + icon +
                '}')
    }
}